migrations/20220105170001-migrate-past-platformTip-orders.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(
+      `
+        UPDATE
+          "Orders"
+        SET
+          "platformTipAmount" = ("data"->'platformFee')::int,
+          "platformTipEligible" = true
+        WHERE "data"->>'isFeesOnTop' = 'true' AND ("data"->'platformFee')::int > 0;
+      `,
+    );
+
+    await queryInterface.sequelize.query(
+      `
+        UPDATE
+          "Orders" as o
+        SET
+          "platformTipAmount" = (o."data"->'platformFee')::int,
```

Comment:
What if platformFee is not set? OrderId=511771 for example.

Reply:
Is `(o."data"->'platformFee')::int` coalesced to `null` ?

Reply:
Exactly. This is valid:
```sql
SELECT NULL::int
```